model_compression_toolkit/core/common/framework_implementation.py

```diff
@@ -382,8 +383,9 @@ def model_grad(self,
             alpha: A tuning parameter to allow calibration between the contribution of the output feature maps returned
                 weights and the other feature maps weights (since the gradient of the output layers does not provide a
                 compatible weight for the distance metric computation).
+            n_iter: The number of random iterations to calculate the approximated jacobian trace for each interest point.
 
-        Returns: A list of normalized gradients to be considered as the relevancy that each interest
+        Returns: A list of normalized jacobians traces to be considered as the relevancy that each interest
```

Comment:
We are computing the trace(jacobians jacobians^T) and not the jacobians traces

Reply:
Is the term "jacobian-based weights" sounds ok? It's not possible to mention  trace(jacobians jacobians^T) everywhere in the comments (it's not just here, and it wouldn't be clear)